fizzbuzz2.rb

```diff
@@ -0,0 +1,13 @@
+range = 1..20
+range.each do |num|
+    if num % 3 == 0 && num % 5 ==0
```

Comment:
後のプラクティスでRubocopを導入すれば機械的に検知できることではありますが一応。

```suggestion
    if num % 3 == 0 && num % 5 == 0
```